templates/00-default.conf

```diff
@@ -79,3 +79,23 @@ port = {{ .PrometheusPort }}
 cafile = {{ .PrometheusCaCertPath }}
 {{ end }}
 {{ end }}
+
+[glance_client]
+endpoint_type=internalURL
+
+[ironic_client]
+endpoint_type=internalURL
+
+[neutron_client]
+endpoint_type=internalURL
+
+[nova_client]
+endpoint_type=internalURL
+
+[placement_client]
+endpoint_type=internalURL
+
+[watcher_cluster_data_model_collectors]
+compute = 900
+baremetal = 900
+storage = 900
```

Comment:
I may be missing something, but I'd say the expected syntax is:

```
[watcher_cluster_data_model_collectors.baremetal]
period = 900

[watcher_cluster_data_model_collectors.compute]
period = 900

[watcher_cluster_data_model_collectors.storage]
period = 900
```

Looking at:

https://docs.openstack.org/watcher/latest/configuration/watcher.html#watcher-cluster-data-model-collectors-compute

and upstream zuul config:

https://github.com/openstack/watcher/blob/master/.zuul.yaml#L49-L54
https://storage.bhs.cloud.ovh.net/v1/AUTH_dcaab5e32b234d56b626f72581e3644c/zuul_opendev_logs_f7d/941026/1/check/watcher-tempest-strategies/f7d455a/controller/logs/etc/watcher/watcher_conf.txt

Reply:
your right...

https://docs.openstack.org/watcher/latest/configuration/watcher.html#watcher-cluster-data-model-collectors-baremetal

we are not ment to have `.` in the name of config options in oslo so that confused me but looking at the 

I should probably add this to the list of tech debt to refactor upstream

Reply:
so these config section are dynmiclly loaded and regeristered form sevador plugins
and the section name is constucted by get_entry_name

https://github.com/openstack/watcher/blame/5f6fbaea56889eaf1bd317f9873f314b716e0ddc/watcher/common/loader/default.py#L64-L65

which is not in line with 

https://docs.openstack.org/oslo.config/latest/reference/naming.html

and genrealy breaks our naming convensions.

when you are namespacing your config sections, it should be done like this:
https://github.com/openstack/os-vif/blob/master/os_vif/plugin.py#L79
i.e. you prefix the namespacen and use _ as the seperator.